cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -53,4 +69,6 @@ package object cwl extends TypeAliases {
 
   type Yaml = String
```

Comment:
I'm personally not a big fan of pretending that `String`s are `Yaml`s. IMO it just gives the false security of type safety while actually just making it harder to work out what arguments are wanted.